### ISSUE TITLE
add option to let H2 send CC/sysex midi messages

### ISSUE THREAD

**Issue by thijz** [hydrogen-music/hydrogen#57]
it would be nice if H2 could send CC midi messages to external apps/hardware
maybe a new type of instrument can be created (a 'CC/Sysex instrument') that can also be added to a drumkit and that allows you to place notes in a pattern just like you place a note on a regular instrument.
the only difference would be that you need some way of entering a totally arbitrary midi message string. A simple 'enter midi message here' popup window would do > it's up to the user to make sure that the midi message he enters is valid.

see http://hydrogen.popez.org/hcms/node/1913#comment-57548
see http://www.hydrogen-music.org/hcms/node/1947#comment-57665

**Comment by thijz** [comment]
similar to ##63 and #195

**Comment by theGreatWhiteShark** [comment]
Nah. I don't like the idea using the notes in the Pattern Editor for anything else than Note On (and Off) events. Maybe the automation track. But no special instrument with special notes.
